Utilities/bootstrap

```diff
@@ -895,14 +895,21 @@ def main():
     # Compute the build paths.
     if platform.system() == 'Darwin':
         build_target = "x86_64-apple-macosx10.10"
-    elif platform.processor() == 's390x':
-        build_target = "s390x-unknown-linux"
-    elif platform.system() == 'Linux' and platform.machine() == 'ppc64le':
-        build_target = 'powerpc64le-unknown-linux'
-    elif platform.system() == 'Linux' and platform.machine() == 'arm':
-        build_target = 'arm-linux-gnueabihf'
+    elif platform.system() == 'Linux':              
+        if platform.machine() == 'x86_64':
+            build_target = "x86_64-unknown-linux"
+        elif platform.machine() == 's390x':
+            build_target = "s390x-unknown-linux"
+        elif platform.machine() == 'ppc64le':
+            build_target = 'powerpc64le-unknown-linux'
+        elif platform.machine().startswith("armv7"):
+            build_target = 'armv7-unknown-linux-gnueabihf'
```

Comment:
Similarly, the previous code has this target triple: `arm-linux-gnueabihf`

Reply:
Yes, 
This change has also been included in https://github.com/uraimo/buildSwiftOnARM/blob/master/swiftpm.diffs/1546.diff
It has been tested on a variety of armv7 boards such as RaspberryPi, Asus Tinkerboard, ODROID Xu4.